src/openforms/forms/tests/admin/test_form.py

```diff
@@ -128,7 +136,8 @@ def test_form_admin_import(self):
         self.assertNotEqual(form.uuid, "b8315e1d-3134-476f-8786-7661d8237c51")
         self.assertEqual(form.name, "Form 000")
         self.assertEqual(form.internal_name, "Form internal")
-        self.assertEqual(form.authentication_backends, ["digid"])
+        self.assertEqual(form.auth_backends.count(), 1)
+        self.assertEqual(form.auth_backends.first().backend, "digid")
```

Comment:
```suggestion
        self.assertEqual(form.auth_backends.get().backend, "digid")
```
avoid the type checker error that `NoneType` doesn't have the attribute `backend`